builders/deploy.yaml

```diff
@@ -178,6 +178,8 @@ steps:
       - --runtime=go113
       - --memory=128
       - --timeout=540s
+      - --vpc-connector projects/erouska-key-server-dev/locations/europe-west1/connectors/serverless-vpc-connector
```

Comment:
I think `--vpc-connector serverless-vpc-connector` should be enough:)
https://cloud.google.com/sdk/gcloud/reference/functions/deploy#--vpc-connector